packages/bus-drivers/README.md

```diff
@@ -1,14 +1,14 @@
 # **🚌 Bus Drivers**
 This folder contains bus drivers for `resolve-es`.
 
-Bus driver is an object that must contain two functions:  
```

Comment:
2 spaces on the end of the string are important because in MD it is replaced by <br/>. I guess you have not prepared vs code for this.